polyphemus/bin/etl_tools

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env ruby
+
+require 'bundler'
+Bundler.require :default, (ENV['POLYPHEMUS_ENV'] || :development).to_sym
+
+require_relative '../lib/polyphemus'
+require 'yaml'
+
+config = YAML.load(File.read(File.expand_path("../../config.yml", __FILE__)))
+
+class RunOrTest < Etna::Command
+  def execute(script_file, record_id)
+    runner = Polyphemus::MagmaEtlScriptRunner.new(script_file)
+    magma_client = Etna::Clients::Magma.new(token: ENV['TOKEN'], host: ENV['MAGMA_HOST'])
+
+    crud = Etna::Clients::Magma::MagmaCrudWorkflow.new(magma_client: magma_client, project_name: runner.project_name)
+    record = crud.lookup_record(runner.model_name, record_id)
+    if record.nil?
+      raise "Could not find #{runner.model_name} by id #{record_id} in project #{runner.project_name}"
+    end
+
+    runner.run(record, magma_client: magma_client, commit: self.commit)
+  end
+
+  def commit
+    false
+  end
+end
+
+class Polyphemus
```

Comment:
I plan to add two more commands to simplify this process:
1.  Generate a new etl script with the right name + template text.
2.  Create a PR to monoetna given an etl script.